test/Microsoft.Health.Fhir.Stu3.Tests.E2E/KnownCrucibleTests.cs

```diff
@@ -429,8 +429,6 @@ public class KnownCrucibleTests
             "testscript-example-update/SETUP",
             "transactionandbatchtest/XFER0",
             "transactionandbatchtest/XFER10",
-            "transactionandbatchtest/XFER11",
-            "transactionandbatchtest/XFER12",
```

Comment:
Got a hint from the crucible test failure as below-
![image](https://user-images.githubusercontent.com/57157506/69983693-2db69800-14ec-11ea-829f-4339c7eaa025.png)